jupyter_server_nbmodel/extension.py

```diff
@@ -56,4 +56,5 @@ def initialize_handlers(self):
     async def stop_extension(self):
         if hasattr(self, "__execution_stack"):
             get_logger().info("Disposing the execution stack…")
+            await self.__execution_stack.dispose()
```

Comment:
This is redundant with the line below it. The wrapping on `wait_for` is to avoid the process to hang; deferring forced clean-up to the Python interpreter.

```suggestion
```